pages.it/windows/robocopy.md

```diff
@@ -0,0 +1,33 @@
+# robocopy
+
+> Copia robusta di file e cartelle.
+> Per impostazione predefinita, i file saranno copiati solo se la sorgente e la destinazione hanno timestamp o dimensioni dei file diversi.
+> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/robocopy>
+
+- Copia tutti i file `.jpg` e `.bmp` da una cartella ad un'altra:
+
+`robocopy {{percorso\alla\cartella\sorgente}} {{percorso\alla\cartella\destinazione}} {{*.jpg}} {{*.bmp}}`
```

Comment:
I don't know Italian, but we prefer something like `percorso\della\directory_sorgente` for `path\to\source_directory` and `path\to\destination_directory`. Please use this format for the rest of the page.